### DISCUSSION TITLE
PV2 Replicate method with no params and NetWork Properties attribute

### DISCUSSION THREAD

**Discussion by duyca** [FirstGearGames/FishNet#615] [Feature Request]
Can we put Replicate data and ReplicateState out side when set it before method run. 
And with network property attribute for Reconcile will be nice.
it will much clearer with PV2 when Replicate and Reconcile will every where.

**Comment by FirstGearGames** [comment]
Those values change at runtime and by having them as parameters it makes it really easy and efficient to work with through FishNet.
You can also change the channel at runtime!

There's no intentions for this change, sorry.
